src/library_glfw.js

```diff
@@ -996,6 +996,13 @@ var LibraryGLFW = {
       }
     },
 
+    defaultWindowHints: () => {
+      GLFW.hints = {};
+      for (var k in GLFW.defaultHints) {
+        GLFW.hints[k] = GLFW.defaultHints[k];
+      }
```

Comment:
I think you can write this as:

``
GLFW.hints = Object.assign({}, GLFW.defaultHints)
```

Reply:
Yes that works and is cleaner. Thanks.